java/arcgis-vectortiledlayer-url/src/main/res/values/strings.xml

```diff
@@ -1,16 +1,18 @@
 <resources>
     <string name="app_name">vector-tiled-layer</string>
 
-    <string name="navigation_url">https://www.arcgis.com/home/item.html?id=e19e9330bf08490ca8353d76b5e2e658</string>
-    <string name="light_gray_url">https://www.arcgis.com/home/item.html?id=5dd75c1a544b46c3af01ba5736bfdfa0</string>
-    <string name="night_url">https://www.arcgis.com/home/item.html?id=92c551c9f07b4147846aae273e822714</string>
-    <string name="dark_gray_url">https://www.arcgis.com/home/item.html?id=5ad3948260a147a993ef4865e3fad476</string>
+    <string name="mid_century_url">https://www.arcgis.com/home/item.html?id=7675d44bb1e4428aa2c30a9b68f97822</string>
+    <string name="colored_pencil_url">https://www.arcgis.com/home/item.html?id=4cf7e1fb9f254dcda9c8fbadb15cf0f8</string>
+    <string name="newspaper_url">https://www.arcgis.com/home/item.html?id=dfb04de5f3144a80bc3f9f336228d24a</string>
+    <string name="nova_url">http://www.arcgis.com/home/item.html?id=75f4dfdff19e445395653121a95a85db</string>
+    <string name="world_street_night_url">https://www.arcgis.com/home/item.html?id=86f556a2d1fd468181855a35e344567f</string>
 
     <string-array name="vector_tiled_types">
-        <item>World Navigation</item>
-        <item>World Street Night</item>
-        <item>Light Gray Canvas</item>
-        <item>Dark Gray Canvas</item>
+        <item>Mid-Century</item>
+        <item>Colored Pencil</item>
+        <item>Newspaper</item>
+        <item>Nova</item>
+        <item>World Street Map (Night)</item>
     </string-array>
 
     <string name="vector_tiled_layer">%1$s</string>
```

Comment:
The two string values below are unused.

Reply:
I saw lint identify that too, but they're use in setupDrawer() at:

https://github.com/Esri/arcgis-runtime-samples-android/blob/5917560b6dd3f7e0401508d584056b5ec90d93ba/java/arcgis-vectortiledlayer-url/src/main/java/com/esri/arcgisruntime/sample/arcgisvectortiledlayerurl/MainActivity.java#L148